docker-compose.yml

```diff
@@ -6,6 +6,11 @@ services:
     ports:
       - '8080:8080'
     volumes:
+      - .:/opt/applepaymerchant
       - ./merchant_identity.pem:/opt/applepaymerchant/merchant_identity.pem
       - ./merchant_identity.key:/opt/applepaymerchant/merchant_identity.key
       - ./config.json:/opt/applepaymerchant/config.json
+  rabbitmq:
+    image: rabbitmq:3.7-alpine
+    ports:
+      - '5672:5672'
```

Comment:
You could consider an `depends_on: applepaymerchant`, however, I bet RabbitMQ starts up quickly enough.

Reply:
Correction: `depends_on: rabbitmq` for `applepaymerchant`, not the other way around.

Reply:
That's a fair point. The only issue is that `depends_on` is rather loose. But since RMQ is probably quick, it is a good idea.